mscore/resourceManager.cpp

```diff
@@ -771,6 +771,7 @@ void ResourceManager::checkPluginUpdate(QPushButton* install)
       if (pluginDescriptionMap.contains(page_url)) {
             auto& desc = pluginDescriptionMap[page_url];
             PluginPackageDescription* desc_tmp = new PluginPackageDescription(desc);
+            install->setText("Checking for update...");
```

Comment:
Needs to be translatable, and use ellipsis "…" rather than 3 dots "...", so `install->setText(tr("Checking for update…"));`

Reply:
Thanks. Fixed in https://github.com/musescore/MuseScore/pull/4833/commits/1d26ad066f4fdc3c5897d2e6c467a6de02ce32f4.